content/index.html

```diff
@@ -11,7 +11,7 @@ <h1 class="text-center">
       <a href="/reference" class="btn btn-outline-white btn-xl">Read the API Reference</a>
     </p>
     <p class="text-center">
-      <a href="https://demo.platforms.bookingsync.com/api-docs/index.html" class="btn btn-outline-white btn-xl">Read Channel API Reference for OTA connection</a>
+      <a href="https://channel-api.developers.bookingsync.com" class="btn btn-outline-white btn-xl">Read Channel API Reference for OTA connection</a>
```

Comment:
Can you please showcase which button is this changing?

Reply:
@pankopiotr I've included that in PR description.